packages/config/config/devices/0x031e/lzw31-bsd_1.47-1.47.json

```diff
@@ -0,0 +1,393 @@
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 15,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "local",
+					"value": 0
+				},
+				{
+					"label": "3-way",
+					"value": 1
+				},
+				{
+					"label": "3-way & local",
+					"value": 3
+				},
+				{
+					"label": "z-wave hub",
+					"value": 4
+				},
+				{
+					"label": "z-wave hub & local",
+					"value": 5
+				},
+				{
+					"label": "z-wave hub & 3-way",
+					"value": 6
+				},
+				{
+					"label": "z-wave hub & local & 3-way",
+					"value": 7
+				},
+				{
+					"label": "timer",
+					"value": 8
+				},
+				{
+					"label": "timer & local",
+					"value": 9
+				},
+				{
+					"label": "timer & 3-way",
+					"value": 10
+				},
+				{
+					"label": "timer & 3-way & local",
+					"value": 11
+				},
+				{
+					"label": "timer & z-wave hub",
+					"value": 12
+				},
+				{
+					"label": "timer & z-wave hub & local",
+					"value": 13
+				},
+				{
+					"label": "timer & z-wave hub & 3-way",
+					"value": 14
+				},
+				{
+					"label": "all",
+					"value": 15
+				}
+			]
+		},
+		"13": {
+			"label": "LED Indicator Color",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 170,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"14": {
+			"label": "LED Indicator Intensity",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"15": {
+			"label": "LED Indicator Intensity (When Off)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"16": {
+			"label": "LED Strip Effect",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 83823359,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"17": {
+			"label": "LED Indicator Timeout",
+			"unit": "seconds",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 3,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"18": {
+			"label": "Active Power Reports",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"19": {
+			"label": "Periodic Power & Energy Reports",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 3600,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"20": {
+			"label": "Energy Reports",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
```

Comment:
It looks like this was propagated from the existing config file, but the LZW31 (Black Series Dimmer) does not support led strip effects (notifications) or energy reporting.

Reply:
https://support.inovelli.com/portal/en/kb/articles/products-switches-dimmer-lzw31-spec-sheet

I noticed it has parameters 17 in their spec sheet.